example/20-componentconfig-gardener-controller-manager.yaml

```diff
@@ -17,6 +17,7 @@ controllers:
   # shootMonitorPeriod: 300s
   shootMaintenance:
     concurrentSyncs: 5
+  # restartControlPlaneControllers: true
```

Comment:
Generally, it just restarts pods with a certain label, independent of whether they are controllers or control plane components. Can we find a more general name? Maybe `enableLabeledPodsRestart` or something similar?

Reply:
> it just restarts pods with a certain label

Imo, this is an implementation detail. The field should be self explainable so that an operator knows what it does. If we just call it something like `labeledPodRestart` then it is quite ambiguous because of: 
a) which labeled Pods?
b) in the shoot, in the seed, or both?

We can rename it but I don't want to be too generic. Maybe something like `enableRestartControlPlaneComponents`. WDYT?

Reply:
What about `enableShootControlPlaneRestarter`?